### DISCUSSION TITLE
Add ProjectDim function

### DISCUSSION THREAD

**Discussion by lzj1769** [GreenleafLab/ArchR#572] [Feature Requests]
Thanks for this great package.

I am wondering if ArchR can provide a function to project matrix to low dimension, 
for example, here is how Seurat does:
```
# If you wold like to observe genes that are strongly correlated with the first MDS coordinate
pbmc <- ProjectDim(pbmc, reduction = "mds")
```

thanks,

**Comment by jgranja24** [comment]
Hmmm this seems reasonable I will take a look further into this.
